plugins/CoreAdminHome/templates/generalSettings.twig

```diff
@@ -10,7 +10,7 @@
     {{ ajax.errorDiv() }}
     {{ ajax.loadingDiv() }}
 
-{% if isGeneralSettingsAdminEnabled %}
+{% if isGeneralSettingsAdminEnabled and not isMultiServerEnvironment %}
```

Comment:
I guess that is actually not needed to hide the archiving settings. They are stored in the database when configured through UI. See https://github.com/matomo-org/matomo/blob/f5e9420a987340b036fa342e876ab92e314f4ec7/core/ArchiveProcessor/Rules.php#L265-L272

Only if that values is not available the one in config should be used.

Hiding the email settings only makes more sense, as they are stored in config only.

Reply:
Thanks @sgiehl, I'll tackle this over the weekend.

Reply:
@sgiehl, I updated the template to hide only the Email Server settings if in multi-server environment.